### PR TITLE
CORE-405 Assimilate pgp_import_keydata (import_key_t)

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
Please review this, @leviathanch, @saschabacardit.

File: src/pgp_sequoia.h [hunk 1]
```diff
@@ -298,8 +298,11 @@ PEP_STATUS pgp_get_key_rating(
  *  @param[in]  changed_key_index    uint64_t *
  *  
  */
-PEP_STATUS pgp_import_keydata(PEP_SESSION session, const char *key_data,
-                              size_t size, identity_list **private_idents,
+PEP_STATUS pgp_import_keydata(PEP_SESSION session,
+                              const char *key_data,
+                              size_t size,
+                              identity_list **idents,
```
Comment (leviathanch) [inline src/pgp_sequoia.h:308]: The parameter is missing in the comment...

**Review by leviathanch — APPROVED** [review]
I'm not picky about the missing entry in the comment, it's just a cosmetic issue.
The changes fix the interface issue.
OK from my side

File: src/pgp_sequoia.h (hunk 1 above)
Comment (ghost) [reply src/pgp_sequoia.h:308]: Cleaned that up, plus added the changelog just in case.

**Review by ip-plank-security — APPROVED** [review]

**Pull request merged**